packages/syntax-core/src/Badge/Badge.module.css

```diff
@@ -0,0 +1,13 @@
+.icon {
+  /* stylelint-disable-next-line declaration-no-important --  Need to use !important to override MUI icon sizes  */
+  height: 16px !important;
+  /* stylelint-disable-next-line declaration-no-important --  Need to use !important to override MUI icon sizes  */
+  width: 16px !important;
+
+  /**
+   * The badge size is 22px total, text is size 14px, there is 4px top + 4px bottom padding
+   * Adjust the margins on the (optional) icon so that is displays the right size but does not affect overall height
+   */
+  margin-top: -1px;
+  margin-bottom: -1px;
```

Comment:
👀 The design calls for badge of always 22px height.  4px top + bottom padding, 14px line-height, 12px font-size, with a 16px w/h Icon size.

The margins here are adjusted by -1px top -1px bottom in order to keep the 16px icon from affecting the overall height of the badge.

Reply:
@benjroy did you confirm with @ajmooo whether this is actually what we want? Ideally the height of the badge would be divisible by 4 - so either `24px` or `20px` high.

Also, the `-1` is a bit of a hack - let's see if we can work with design to avoid it

Reply:
Thank you!  Have not yet confirmed that this is what we actually want.  The `-1` is definitely a hack, and your review is just what I need to give a poke to update the `Badge` in figma!